README.md

```diff
@@ -193,3 +196,35 @@ Use this with any client interface such as [MoveIt!](https://moveit.ros.org/) or
 rosrun rqt_joint_trajectory_controller rqt_joint_trajectory_controller
 ```
 
+## Troubleshooting
```

Comment:
Suggestion: make each Q&A a separate sub section (instead of a numbered bullet).

It would clean up the markdown a bit, and reduces the need for the embedded html.

Reply:
That's a very good suggestion, thank you.